denonavr/api.py

```diff
@@ -11,8 +11,10 @@
 import logging
 import xml.etree.ElementTree as ET
 
+import time
 from io import BytesIO
-from typing import Awaitable, Callable, Dict, Hashable, Optional, Tuple
+from typing import cast, Awaitable, Callable, Dict, Hashable, Optional, Tuple
+import async_timeout
```

Comment:
In core, we prefer `async_timeout` as it doesn't create a task which reduces the overhead.  It will be an asyncio built-in in python 3.11+

```
if sys.version_info[:2] < (3, 11):
    from async_timeout import timeout as asyncio_timeout
else:
    from asyncio import timeout as asyncio_timeout
```

Reply:
Some additional background: we have slowly been replacing asyncio.wait_for in core because there were so many tasks being created from all the various uses by integrations that it led to increased event loop latency and made HA less responsive. We now include measuring the number of tasks for minute when we get performance issues opened for core from profiles. This effort (combined with other efforts to rescue tasks) has been largely successful the average number of tasks per minute has dropped significantly and we have a lot fewer latency complaints these days.

Reply:
okay, then let's try it like this 😄 

```suggestion
if sys.version_info[:2] < (3, 11):
    from async_timeout import timeout as asyncio_timeout
else:
    from asyncio import timeout as asyncio_timeout
```

Reply:
Resolved in https://github.com/dcmeglio/denonavr/commit/2362b017c3c80bc993a3e753f246fcb4665cea62